teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -221,12 +221,13 @@ interface VcsRoot {
     val id: VcsRootId
     val name: String
 
-    fun fetchVcsRootProperties(): List<VcsRootProperty>
+    fun getUrl(): String?
+    fun getDefaultBranch(): String?
 }
 
 interface VcsRootProperty {
```

Comment:
Do we still need `VcsRootProperty` interface in the API? Looks like it can be moved to `implementation.kt`. Also name `VcsRootProperty` looks too specific for an interface which contains properties `name` and `value` only.

Reply:
No, we don't need the interface `VcsRootProperty` within the API. I removed it and included the class `NameValueProperty` to `implementation.kt`.

Additionally, I created the function `getNameValueProperty()` to extract a certain value of a `List<NameValueProperty>` determined by the `name`. The `VcsRoot` now fetches the properties and maps them to such a list. This function can be used for any type of object having a property list.

Feel free to give better naming suggestions for the `NameValueProperty`.